### PR TITLE
Rename @uifabric/react-hooks & @uifabric/utilities

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [x] Addresses an existing issue: Part of #13384
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Rename @uifabric/react-hooks to @fluentui/react-hooks & @uifabric/utilities to @fluentui/utilities

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by ecraig12345** [comment]
Even once the merge conflicts are resolved, the build for this PR is still going to fail on `projects-test`. @xugao's PR #15620 has a workaround for that and will hopefully be merged Wednesday morning.

File: yarn.lock [hunk 1]
```diff
@@ -1384,22 +1384,6 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
-"@fluentui/date-time-utilities@^7.9.0":
```
Comment (xugao) [inline yarn.lock:1387]: this is wrong. you need to update the package names in root `package.json` file as well

File: change/@fluentui-foundation-legacy-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-foundation-legacy-2020-10-20-18-13-15-hooks-utilities-rename.json:2]: check change failed. you need to update the change type for `-beta` packages to be `prerelease`

Comment (xugao) [inline change/@fluentui-foundation-legacy-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-merge-styles-2020-10-21-11-45-46-hooks-utilities-rename.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-merge-styles-2020-10-21-11-45-46-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-monaco-editor-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-monaco-editor-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-button-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-button-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-cards-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-cards-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-checkbox-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-checkbox-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-docsite-components-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 9]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-docsite-components-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-experiments-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 10]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-experiments-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-focus-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 11]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-focus-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-hooks-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 12]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-hooks-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-icons-mdl2-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 13]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-icons-mdl2-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-internal-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 14]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-internal-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-link-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 15]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-link-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-monaco-editor-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 16]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-monaco-editor-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-slider-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 17]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-slider-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-tabs-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 18]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-tabs-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-theme-provider-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 19]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-theme-provider-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-react-toggle-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 20]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-react-toggle-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-style-utilities-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 21]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-style-utilities-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-theme-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 22]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-theme-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@fluentui-utilities-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 23]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@fluentui-utilities-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@uifabric-date-time-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 24]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@uifabric-date-time-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@uifabric-merge-styles-2020-10-20-18-13-15-hooks-utilities-rename.json [hunk 25]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (xugao) [inline change/@uifabric-merge-styles-2020-10-20-18-13-15-hooks-utilities-rename.json:2]:
```suggestion
  "type": "prerelease",
```

**Review by xugao — CHANGES_REQUESTED** [review]

**Review by ecraig12345 — APPROVED** [review]

**Review by xugao — APPROVED** [review]

**Pull request merged**